packages/auth/src/types/index.ts

```diff
@@ -17,13 +18,16 @@ export type { AuthStandardAttributeKey } from './models/AuthStandardAttributeKey
 export type { AuthUserAttributeKey } from './models/AuthUserAttributeKey';
 export type { AuthUserAttribute } from './models/AuthUserAttribute';
 export type { GetAttributeKey } from './models/GetAttributeKey';
+export type { AuthNextResetPasswordStep } from './models/AuthNextResetPasswordStep';
 
 // Options
 export type { AuthServiceOptions } from './options/AuthServiceOptions';
 export type { AuthSignUpOptions } from './options/AuthSignUpOptions';
 
 // Requests
+export type { ResetPasswordRequest } from './requests/ResetPasswordRequest';
 export type { SignUpRequest } from './requests/SignUpRequest';
 
 // Results
 export type { AuthSignUpResult } from './results/AuthSignUpResult';
+export type { ResetPasswordResult } from './results/ResetPasswordResult';
```

Comment:
Didn't catch this on the other PR, but I wonder if the type aliases are really required here since these are our types and the names are the same. 

```suggestion
export { AuthUserAttributeKey } from './models/AuthUserAttributeKey';
export { AuthUserAttribute } from './models/AuthUserAttribute';
export { GetAttributeKey } from './models/GetAttributeKey';
export { AuthNextResetPasswordStep } from './models/AuthNextResetPasswordStep';

// Options
export { AuthServiceOptions } from './options/AuthServiceOptions';
export { AuthSignUpOptions } from './options/AuthSignUpOptions';

// Requests
export { ResetPasswordRequest } from './requests/ResetPasswordRequest';
export { SignUpRequest } from './requests/SignUpRequest';

// Results
export { AuthSignUpResult } from './results/AuthSignUpResult';
export { ResetPasswordResult } from './results/ResetPasswordResult';
```

For some reason GitHub won't let me comment on the entire list, but the others above my suggestion can probably be changed as well.